src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -33,6 +34,15 @@ import type {BaseSelectionListProps, ButtonOrCheckBoxRoles, FlattenedSectionsRet
 
 const getDefaultItemHeight = () => variables.optionRowHeight;
 
+type SelectionListWrapperProps = {children: ReactNode; shouldWrapSectionList: boolean};
```

Comment:
This is a fix related to a bug with scrolling. It's happening due to nested `SectionList` - this shouldn't be happening because we're inside a Modal but for some reason it does.

I will create a follow-up PR with a proper fix, I don't want to delay this feature from being done with such an issue. 🙌🏻